server/protocols/jmap-rfc-8621/pom.xml

```diff
@@ -37,6 +37,11 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-jmap</artifactId>
+            <version>${project.version}</version>
```

Comment:
We should have the version in main pom, so that we don't need to specify this here and in jmap-draft POM